src/groups/mqb/mqbblp/mqbblp_relayqueueengine.cpp

```diff
@@ -74,6 +74,10 @@ const bsls::Types::Int64 k_NS_PER_MESSAGE =
     bdlt::TimeUnitRatio::k_NANOSECONDS_PER_MINUTE / k_MAX_INSTANT_MESSAGES;
 // Time interval between messages logged with throttling.
 
+#define LOGTHROTTLE_INFO()                                                    \
+    BALL_LOGTHROTTLE_INFO(k_MAX_INSTANT_MESSAGES, k_NS_PER_MESSAGE)           \
+        << "[THROTTLED] "
```

Comment:
Overall, what do you thing about defining this macro in one of our helper headers, with an unique prefix?

Do we actually need to make these constants `k_MAX_INSTANT_MESSAGES`/`k_NS_PER_MESSAGE` different?